packages/clerk-js/src/core/fapiClient.ts

```diff
@@ -227,7 +227,7 @@ export function createFapiClient(options: FapiClientOptions): FapiClient {
     // avoid triggering a CORS OPTION request as it currently breaks cookie dropping in Safari.
     const overwrittenRequestMethod = method === 'GET' ? 'GET' : 'POST';
     let response: Response;
-    const urlStr = requestInit.url.toString();
+    const url = requestInit.url;
```

Comment:
We were converting the url to string here, then converting it back to a url for modification and then converting it a string.  Now we are just converting it to string once and passing the URL object to fetch